specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-11-16-preview/authorization-AccessReviewCalls.json

```diff
@@ -0,0 +1,2665 @@
+        "x-ms-examples": {
+          "GetAccessReviewHistoryDefinition": {
+            "$ref": "./examples/GetAccessReviewHistoryDefinition.json"
+          }
+        },
+        "description": "Get access review history definition by definition Id",
+        "parameters": [
+          {
+            "name": "historyDefinitionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the access review history definition."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "AccessReviewHistoryDefinition",
+            "schema": {
+              "$ref": "#/definitions/AccessReviewHistoryDefinition"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorDefinition"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PutAccessReviewHistoryDefinition"
+        ],
+        "operationId": "AccessReviewHistoryDefinition_Create",
+        "x-ms-examples": {
+          "PutAccessReviewHistoryDefinition": {
+            "$ref": "./examples/PutAccessReviewHistoryDefinition.json"
+          }
+        },
+        "description": "Create a scheduled or one-time Access Review History Definition",
+        "parameters": [
+          {
+            "name": "historyDefinitionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the access review history definition."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "AccessReviewHistoryDefinition",
+            "schema": {
+              "$ref": "#/definitions/AccessReviewHistoryDefinition"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorDefinition"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "PatchAccessReviewHistoryDefinition"
+        ],
+        "operationId": "AccessReviewHistoryDefinition_GenerateDownloadUri",
+        "x-ms-examples": {
+          "PatchAccessReviewHistoryDefinition": {
+            "$ref": "./examples/PatchAccessReviewHistoryDefinition.json"
+          }
+        },
+        "description": "Generates a uri which can be used to retrieve review history data. This URI has a TTL of 1 day and can be retrieved by fetching the accessReviewHistoryDefinition object.",
```

Comment:
>Generates a uri which can be used to retrieve review history data [](http://example.com/codeflow?start=24&length=65)

Is this accurate?

Reply:
> 

Yes this is accurate. For customers to fetch their history we generate a SAS link for the user specifically which expires after an hour. We return the URI to the user so they can use it to download the results.